bigquery/google/cloud/bigquery/query.py

```diff
@@ -31,7 +31,6 @@ class _SyncQueryConfiguration(object):
     Values which are ``None`` -> server defaults.
     """
     _default_dataset = None
-    _dry_run = None
```

Comment:
I thought that entire ["request body"](https://cloud.google.com/bigquery/docs/reference/rest/v2/jobs/query#request-body) was the `['configuration']` object. I think I'm lost in which levels are where entirely.